packages/cli/src/commands/setup/auth/templates/auth.ts.template

```diff
@@ -1,6 +1,13 @@
 import { parseJWT } from '@redwoodjs/api'
 import { AuthenticationError, ForbiddenError } from '@redwoodjs/graphql-server'
 
+/**
+ * Represents the user attributes returned by the decoding the
+ * Authentication provider's JWT together with an optional list of roles.
+ */
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+type RedwoodUser = Record<string, any> & { roles?: string[] }
```

Comment:
I would use unknown here,instead of any

Reply:
Will do. That means I can also remove the lint disable.